views/config.php

```diff
@@ -292,491 +57,8 @@
             <label>REST API Key<i class="tiny circular help icon link" role="popup" data-title="Rest API Key" data-content="Your 48 character alphanumeric REST API Key. You can find this in App Settings > Keys & IDs." data-variation="wide"></i></label>
             <input type="text" name="app_rest_api_key" placeholder="xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" value="<?php echo esc_attr(OneSignal::maskedRestApiKey($onesignal_wp_settings['app_rest_api_key'])); ?>">
           </div>
-          <div class="field subdomain-feature">
```

Comment:
This is a breaking change in a patch version. Things should keep working as-is for 2.x versions, so we should allow both viewing and editing of these settings.
* If the customer follows the migration guide, then settings on dashboard.onesignal.com will override these, so optionally we put a warning about that here.